test/search_tests.cpp

```diff
@@ -96,6 +96,88 @@ BOOST_AUTO_TEST_CASE( get_one_objects ) {
     free_client(client);
 }
 
+BOOST_AUTO_TEST_CASE(get_objects_with_plus_query_param) {
+
+        printf("-----Testing Search Object with +-------\n");
+	uint64_t num_objs;
+	ds3_client* client = get_client();
+	ds3_get_objects_response* response;
+	const char* bucket_name = "get_objects_with_plus_query_param";
+	const char* plus_object = "Plus+Object";
+
+        ds3_request* put_bucket_request = ds3_init_put_bucket(bucket_name);
```

Comment:
Ahh, got it . . . I used spaces instead of tabs.  It looks fine on mine.  I'll change to tabs to match the rest of the file.